nautilus_trader/adapters/interactive_brokers/parsing/instruments.py

```diff
@@ -74,8 +76,16 @@
     "NYBOT",  # US
     "SNFE",  # AU
 ]
+VENUES_CFD = [
```

Comment:
@DracheShiki all of these are translating to `SMART`? If so just use one i.e `IBCFD` to keep things simple.

Reply:
Understood, lets leave it the way you did.